fbgemm_gpu/experimental/gen_ai/src/quantize/ck_extensions/fp8_blockwise_gemm.hip

```diff
@@ -12,11 +12,14 @@
 #include <numeric>
 
 #include <ATen/ATen.h>
-#include <c10/hip/HIPStream.h>
+#if !defined(USE_ROCM)
```

Comment:
is this block even needed? If USE_ROCM is not defined, it'll basically be an empty file. So we can just remove this block here.

Reply:
Let's just remove this include and move it down to the USE_ROCM. No need to include cuda header here.

Reply:
@xw285cornell  @jianyuh 

changed as required. Please review.